src/datastore/helpers.ts

```diff
@@ -898,3 +898,18 @@ export function registerMempoolPromStats(pgEvents: PgStoreEventEmitter) {
     });
   });
 }
+
+export function convertTxQueryREsultToDbMempoolTx(txs: TxQueryResult[]): DbMempoolTx[] {
+  const dbMempoolTxs: DbMempoolTx[] = [];
+  for (const tx of txs) {
+    const dbMempoolTx: DbMempoolTx = Object.assign(tx, {
+      pruned: false,
+      receipt_time: Math.round(Date.now() / 1000),
+      fee_rate: BigInt(tx.fee_rate),
+      token_transfer_amount: BigInt(tx.token_transfer_amount ?? 0),
+      sponsor_address: tx.sponsor_address ?? undefined,
```

Comment:
This is needed. Otherwise:

Type 'TxQueryResult & { pruned: boolean; receipt_time: number; fee_rate: bigint; token_transfer_amount: bigint; sponsor_address: string | null; }' is not assignable to type 'DbMempoolTx'.
  Types of property 'sponsor_address' are incompatible.
    Type 'string | null' is not assignable to type 'string | undefined'.
      Type 'null' is not assignable to type 'string | undefined'.ts(2322)